api/src/main/resources/workbench.yaml

```diff
@@ -1207,6 +1224,40 @@ definitions:
   RegistrationRequest:
     type: object
 
+  IdVerificationRequest:
+    type: object
+    properties:
+      firstName:
+        type: string
+      lastName:
+        type: string
+      streetLine1:
+        type: string
+      streetLine2:
+        type: string
+      city:
+        type: string
+      state:
+        type: string
+      zip:
+        type: string
+      dob:
```

Comment:
Should this be a date instead?

Reply:
Seems like dates are problematic in Swagger. Also, it feels weird to me to specify a type that doesn't exist in JSON.